examples/nextjs-with-typescript/pages/MuxPlayerTheme.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import Head from 'next/head';
+import { useRef, useState } from "react";
+import MuxPlayer, { MuxPlayerProps } from "@mux/mux-player-react";
+import "media-chrome/dist/themes/media-theme-youtube";
```

Comment:
question(non-blocking): Does this order matter anymore? Assuming not based on the order of imports, but just confirming. Also def fine if it does for this pass regardless.

Reply:
indeed order doesn't matter anymore 👍